File: Modul 3/PRAK302-2410817110020-AkmalRizki.py
nilai = int(input())

if nilai >= 80:
    hasil = 'A'
elif nilai >= 70 and nilai < 80:
    hasil = 'B'
elif nilai >= 60 and nilai < 70:
    hasil = 'C'
elif nilai >= 50 and nilai < 60:
    hasil = 'D'
else:
    hasil = 'E'
print(hasil)

File: Modul 3/PRAK302-2410817110020-AkmalRizki.c
#include <stdio.h>

int main() {
    int nilai;

    scanf("%d", &nilai);

    if (nilai >= 80) {
        printf("A\n");
    } 
    else if (nilai >= 70) {
        printf("B\n");
    } 
    else if (nilai >= 60) {
        printf("C\n");
    } 
    else if (nilai >= 50) {
        printf("D\n");
    } 
    else if (nilai < 50) {
        printf("E\n");
    } 
    else {
        printf("Nilai tidak valid!\n");
    }

    return 0;
}
